### PR TITLE
Volunteer Section

### LINKED ISSUE #134: Update Volunteer section
Volunteer section currently works as css grid, but need to confirm the design spacing, and set up the mobile grid areas

### PR DESCRIPTION
**Pull request by cheshireoctopus**
## Overview

Implements the "Volunteer" section of the CFD website as described in the [Figma](https://www.figma.com/file/KwZFEaLN4u9Uids0Xjjt2J/Code-For-Denver-Website-Redesign-'20?node-id=832%3A430) for desktop and mobile :

image

### CODE REVIEW

File: _sass/footer.scss [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 footer {
 	color: $navy;
+
```
Comment (cheshireoctopus) [inline _sass/footer.scss:5]: Had to scope these styles within `footer` as they were impacting global scope.

Realizing now this file is indented with tabs while the other `.scss` files are indented with spaces. Should I use this PR to change the indentation here?

File: _sass/main.scss [hunk 2]
```diff
@@ -131,6 +131,22 @@ p {
   color: $navy;
 }
```
Comment (cheshireoctopus) [inline _sass/main.scss:133]: Okay to add these styles to the global scope?

Here is how the `a` renders:

image

and here is how `.button-link` renders:

image

File: _sass/footer.scss (hunk 1 above)
Comment (rhyn0bytes) [reply _sass/footer.scss:5]: I think added this change on this PR will be fine. I will send back with this request.

File: _sass/main.scss (hunk 2 above)
Comment (rhyn0bytes) [reply _sass/main.scss:133]: Yes, adding that style to the global scope is perfect.

**Review by rhyn0bytes — APPROVED** [review]
Looks like the styles have been updated already. I think this is perfect to merge.

**Pull request merged**